suite-native/discovery/src/discoveryConfigSlice.ts

```diff
@@ -86,23 +88,29 @@ export const selectAreTestnetsEnabled = (state: DiscoveryConfigSliceRootState) =
 export const selectDiscoveryInfo = (state: DiscoveryConfigSliceRootState) =>
     state.discoveryConfig.discoveryInfo;
 
+export const selectFeatureFlagEnabledNetworkSymbols = memoize((state: FeatureFlagsRootState) => {
```

Comment:
All these selectors are introduced only because feature flags?

Reply:
yes

Reply:
Are we going to revert these changes once we remove that feature flags?

Reply:
we will probably keep it there for Solana and remove it right after that is released

Reply:
ok then it's fine